### DISCUSSION TITLE
Device search

### DISCUSSION THREAD

**Discussion by Bundyra** [frangoteam/FUXA#950] [Ideas]
It would be nice to have the ability to search for devices, just like it is possible with tags.
![image](https://github.com/frangoteam/FUXA/assets/79423733/8f5e0b9f-893e-4513-aec5-132d9a2f3959)
![image](https://github.com/frangoteam/FUXA/assets/79423733/c49b9c53-3285-4ddf-9d65-2cf44da299b6)

**Comment by unocelli** [comment]
Just for curiosity, how many devices and what type have you already configured in a project?

**Comment by Bundyra** [comment]
Its about 50 i guess, cant count them right now. Mostly modbusTCP.
